lib/rubocop/cop/rspec/single_argument_message_chain.rb

```diff
@@ -0,0 +1,53 @@
+module RuboCop
+  module Cop
+    module RSpec
+      # Checks that chains of messages contain more than one element.
+      #
+      # @example
+      #   # bad
+      #   allow(foo).to receive_message_chain(:bar).and_return(42)
+      #
+      #   # good
+      #   allow(foo).to receive(:bar).and_return(42)
+      #
+      #   # also good
+      #   allow(foo).to receive(:bar, :baz)
+      #   allow(foo).to receive("bar.baz")
+      #
+      class SingleArgumentMessageChain < Cop
+        MESSAGE = 'Use `%<recommended_method>s` instead of calling ' \
+          '`%<called_method>s` with a single argument'.freeze
+
+        MESSAGE_CHAIN_METHODS = [
+          :receive_message_chain,
+          :stub_chain
+        ].freeze
```

Comment:
I meant something like adding

```diff
diff --git i/lib/rubocop/rspec/language.rb w/lib/rubocop/rspec/language.rb
index d44e2b7..be2b92f 100644
--- i/lib/rubocop/rspec/language.rb
+++ w/lib/rubocop/rspec/language.rb
@@ -31,6 +31,10 @@ module RuboCop
         attr_reader :selectors
       end

+      module Matchers
+        MESSAGE_CHAIN = SelectorSet.new(%i(receive_message_chain stub_chain))
+      end
+
       module ExampleGroups
         GROUPS  = SelectorSet.new(%i(describe context feature example_group))
         SKIPPED = SelectorSet.new(%i(xdescribe xcontext xfeature))
```

to https://github.com/backus/rubocop-rspec/blob/master/lib/rubocop/rspec/language.rb

`Matchers::MESSAGE_CHAIN.include?(method_name)` below

For context: I added this language file because most of the cops duplicated knowledge about part of the RSpec public API and many cops had different holes depending on what the original cop author remembered to use. Unifying the language definitions made it easier to make sure that all cops were in sync